ccd-definition/ComplexTypes/Recipients.json

```diff
@@ -0,0 +1,93 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "name",
+    "FieldType": "Text",
+    "ElementLabel": "Name of recipient",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "addressCheck",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Do you have the recipient's address?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "address",
+    "FieldType": "AddressUK",
+    "ElementLabel": "Recipient's address",
+    "HintText": "Enter a UK postcode",
+    "FieldShowCondition": "addressCheck=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "documents",
+    "FieldType": "TextArea",
+    "ElementLabel": "Documents",
+    "HintText": "For example, standard directions order or c6",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "date",
+    "FieldType": "Date",
+    "ElementLabel": "Date sent",
+    "HintText": "For example, 21 9 2019",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "timeSent",
+    "FieldType": "Text",
+    "ElementLabel": "Time sent",
+    "HintText": "For example, 2:30pm",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "sentBy",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "StatementOfServiceSentBy",
```

Comment:
Do we need the 'StatementOfService' text here? Makes it less reusable.

Reply:
good shout :)